app/bundles/CampaignBundle/Controller/Api/CampaignApiController.php

```diff
@@ -357,4 +361,103 @@ public function cloneCampaignAction($campaignId)
 
         return $this->handleView($view);
     }
+
+    /**
+     * Get a list of events.
+     *
+     * @return Response
+     */
+    public function exportCampaignAction(Request $request, int $campaignId)
+    {
+        // Check if the campaign exists
+        $campaign = $this->model->getEntity($campaignId);
+        if (!$campaign) {
+            return $this->notFound();
+        }
+
+        // Check if user has permission to export campaigns
+        if (!$this->security->isAdmin() && !$this->security->isGranted('campaign:export:enable', 'MATCH_ONE')) {
```

Comment:
The isAdmin check is already part of the isGranted check so it can be removed, right?
```suggestion
        if (!$this->security->isGranted('campaign:export:enable', 'MATCH_ONE')) {
```

Reply:
fixed